ACM_General/core/templates/core/base.html

```diff
@@ -1,85 +1,52 @@
-<!DOCTYPE html>
-{% load static %}
-
-{% block html_tag %}<html lang='en'>{% endblock %}
+<!DOCTYPE html>{% load static %}
+{% block html_tag %}
+  <html lang='en'>{% endblock %}
```

Comment:
Would there be any way to have the block start and end in a more stylish way. I understand from a html perspective 
this makes more sense with the formatting, but doesn't look good.